demos/coreMQTT/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# AFR coreMQTT demo
+afr_demo_module(core_mqtt)
+
+# Set the default demo that is enabled in the downloaded package.
+afr_set_demo_metadata(ID "CORE_MQTT_DEMO")
```

Comment:
So OCW will take this ID and prepend "CONFIG_" and append "_ENALBED", so we have to choose one to be default.

Reply:
I put plaintext as the default for now